replace_nks.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
we may not need this file in the current directory

Reply:
ok, we have deleted it